src/api/code-challenge/content-types/code-challenge/curriculum-helpers.ts

```diff
@@ -0,0 +1,30 @@
+// @ts-expect-error will fix later
```

Comment:
Can you remove whichever functions are unused?

Reply:
Done